evalml/automl/automl_search.py

```diff
@@ -908,16 +938,15 @@ def full_rankings(self):
 
     @property
     def best_pipeline(self):
-        """Returns an untrained instance of the best pipeline and parameters found during automl search.
+        """If `train_best_pipeline`, returns a trained instance of the best pipeline and parameters found during automl search. Otherwise, returns an untrained pipeline instance.
```

Comment:
This is good. I think it could be easier to follow if you said "Returns an untrained instance of the best pipeline and parameters found during automl search. If `train_best_pipeline` is set to False, returns an untrained pipeline instance.